innovation.game.php

```diff
@@ -7300,6 +7300,22 @@ function stPlayerInvolvedTurn() {
                 self::setGameStateValue('card_id_3', self::executeDraw($player_id, $top_green_card_age, 'hand')['id']);
                 $step_max = 2; // --> 2 interactions: see B
                 break;
+				
+            // id 117, Artifacts age 1: Electrum Stater of Efesos
+            case "117N1":
+                while(true) {
+                    $card = self::executeDraw($player_id, 3, 'revealed'); // "Draw and reveal a 1"
+                    $top_card = self::getTopCardOnBoard($player_id, $card['color']);
+                    if ($top_card == null) 
+					{ // "If you do not have a top card of the drawn card's color"
+                        self::transferCardFromTo($card, $player_id, 'board', false, true); // "meld it"
```

Comment:
Please drop the last two parameters:

self::transferCardFromTo($card, $player_id, 'board');

Setting the last parameter to true actually makes this transfer count as a "score" towards the monument achievement.

Reply:
done